marklogic-data-hub-spark-connector/gradle.properties

```diff
@@ -5,3 +5,4 @@ sparkVersion=2.4.5
 #Properties to be used for testing locally using gradle
 mlHost=localhost
 isDhs=false
+version=2.4.5.0
```

Comment:
Can you put this at the top of the file? It should not be in a section with a comment above of it "Properties to be used for testing locally".

Reply:
done